src/protocol-parser/generated/webdriver-bidi.ts

```diff
@@ -282,7 +287,7 @@ export namespace Session {
           platformName: z.string(),
           setWindowRect: z.boolean(),
           proxy: Session.ProxyConfigurationSchema.optional(),
-          webSocketUrl: z.string().optional(),
+          webSocketUrl: z.union([z.string(), z.literal(true)]).optional(),
```

Comment:
This looks like regression? The Spec defines it as string only atm. Please verify that you are using the latest version of the CDDL

Reply:
> This looks like regression? The Spec defines it as string only atm. Please verify that you are using the latest version of the CDDL

It's generated from this commit, which is the latest one: https://github.com/w3c/webdriver-bidi/commit/6ebacb6553f77fef5a7d29d6e7131bb92f9146ba

Reply:
Ooops. You are right, it was done from the wrong commit )

Reply:
Fixed.